js/app.js

```diff
@@ -33,39 +35,56 @@ class Player {
     }
     restart(posX, posY) {
         if ((this.x >= posX - 50 && this.x <= posX + 50) && (this.y >= posY - 50 && this.y <= posY + 50)) {
-            this.x = 200;
-            this.y = 400;
+            document.getElementById('score').textContent = score;
+            document.getElementById('game-over').classList.remove('invisible');
+            document.getElementById('game-over').classList.add('visible');
+            gameStatus = 'pause';
+            score = 0;
         }
     }
     handleInput(key) {
         switch (key) {
             case 'left': {
-                if (this.x > 0) {
+                if (this.x > 0 && gameStatus === 'start') {
```

Comment:
we should have a constant with the value 'start', like cons START_STATUS = 'start';